### DISCUSSION TITLE
How to use mockoon to mock API when I am not targeting localhost

### DISCUSSION THREAD

**Discussion by ndaian** [mockoon/mockoon#707] [General]
I am new to mockoon so I have a newbie question.
Our app targets the API on a DEV environment, something like http://10.10.20.5:9094/api/riskmapper. Is it possible to specify a base-URL in mockoon? I would like "http://10.10.20.5:9094" to be the base URL instead of "localhost:3000". If that is not possible I would need to change the application's code to target localhost and I read somewhere in the documentation that we do not need to change anything in code.

**Comment by 255kb** [comment]
I think the most straightforward solution would be to make changes to your host file.
Another solution is to deploy your mock on the DEV environment using the [CLI](https://mockoon.com/cli/). Then you would be able to call the dev environment to reach Mockoon CLI. 
But there is no feature allowing mockoon to be served on a different IP than a local network adapter (127.0.0.1, 192.168.x.x, etc.)
